python_modules/dagster/dagster/core/storage/schedules/sql_schedule_storage.py

```diff
@@ -140,7 +140,7 @@ def get_ticks(self, origin_id, before=None, after=None, limit=None):
             db.select([JobTickTable.c.id, JobTickTable.c.tick_body])
             .select_from(JobTickTable)
             .where(JobTickTable.c.job_origin_id == origin_id)
-            .order_by(JobTickTable.c.id.desc())
+            .order_by(JobTickTable.c.timestamp.desc())
```

Comment:
actually, should we do this, or should we add an index on id?

I feel confused about what we should do between this and https://github.com/dagster-io/dagster/pull/6620

Reply:
yeah, exactly.

but it looks like we already have an index on event log, for ID (with an event_type):
```
db.Index(
    "idx_event_type",
    SqlEventLogStorageTable.c.organization_id,
    SqlEventLogStorageTable.c.deployment_id,
    SqlEventLogStorageTable.c.dagster_event_type,
    SqlEventLogStorageTable.c.id,
)
```
but we have it by timestamp for ticks:
```
db.Index(
    "idx_job_tick_timestamp",
    JobTickTable.c.organization_id,
    JobTickTable.c.deployment_id,
    JobTickTable.c.job_origin_id,
    JobTickTable.c.timestamp,
)
```

So, I guess based on the existing indexes on the respective tables (ticks, event_log), this change makes sense.